lib/MiLight/CctPacketFormatter.cpp

```diff
@@ -37,11 +54,24 @@ void CctPacketFormatter::updateTemperature(uint8_t value) {
 }
 
 void CctPacketFormatter::command(uint8_t command, uint8_t arg) {
```

Comment:
I think it's probably fine to have it here, but there's a default-noop method in `PacketFormatter` called `finalizePacket`.  I think it'd make sense to override this method and calculate the checksum there.  [Here's an example](https://github.com/sidoh/esp8266_milight_hub/blob/master/lib/MiLight/V2PacketFormatter.cpp#L54-L56) of it being used in the 9-byte protocol formatter.

Reply:
Yes, your suggestion would be a better solution. I'll try this later.